docker/lr-winnowmap/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:18.04
+
+RUN apt-get -qqy update && \
+    apt-get -qqy dist-upgrade && \
+    apt-get install -y make gcc g++ libz-dev wget xz-utils git
+
+
+RUN wget https://github.com/marbl/meryl/releases/download/v1.3/meryl-1.3.Linux-amd64.tar.xz && \
+    mv meryl-1.3.Linux-amd64.tar.xz /opt && \
+    cd /opt && \
+    tar -xJf meryl-1.3.Linux-amd64.tar.xz
```

Comment:
A good practice is to have an "environment" variable that specifies version, so that if later we decide to update to another version, fewer lines need to be touched hence this is less human-error-prune.